pygmt/src/solar.py

```diff
@@ -0,0 +1,91 @@
+"""
+solar - Plot day-night terminators.
+"""
+
+from pygmt.clib import Session
+from pygmt.helpers import (
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@use_alias(
+    R="region",
+    J="projection",
+    G="fill",
+    B="frame",
+    Td="day_night",
+    Tc="civil_twilight",
+    Tn="nautical_twilight",
+    Ta="astronomical_twilight",
+    W="pen",
+    U="timestamp",
+    V="verbose",
+    X="xshift",
+    Y="yshift",
+    p="perspective",
+)
+@kwargs_to_strings(R="sequence", p="sequence")
+def solar(self, **kwargs):
+    r"""
+    A function that plots the day-night terminator. Additionally, it can
+    plot the terminators for civil twilight, nautical twilight, and
+    astronomical twilight.
+
+    {aliases}
+
+    Parameters
+    ----------
+    {R}
+    {J}
+    {B}
+    fill : str
+        *fill color*\ [**@**\ *transparency*]
+        The fill color for the night-area plotted. The optional
+        transparency parameter can be appended after the color
+        using **@** after the color.
```

Comment:
@transparency is a general feature for all colors, and may not be suitable to put it here.
```suggestion
        Color or pattern for filling of terminators.
```